packages/core/src/app.ts

```diff
@@ -144,6 +145,39 @@ export function excludeDotFiles(files: string[]) {
   return files.filter(file => !file.startsWith('.') && !file.includes('/.'));
 }
 
+export const CACHE_BUSTING_PLUGIN = {
+  path: require.resolve('./babel-plugin-cache-busting'),
+  version: readJSONSync(`${__dirname}/../package.json`).version,
+};
+
+export function addCachablePlugin(babelConfig: TransformOptions) {
+  if (Array.isArray(babelConfig.plugins) && babelConfig.plugins.length > 0) {
+    const plugins = Object.create(null);
+    plugins[CACHE_BUSTING_PLUGIN.path] = CACHE_BUSTING_PLUGIN.version;
+
+    for (const plugin of babelConfig.plugins) {
+      let absolutePathToPlugin: string;
+      if (Array.isArray(plugin) && typeof plugin[0] === 'string') {
+        absolutePathToPlugin = plugin[0] as string;
+      } else if (typeof plugin === 'string') {
+        absolutePathToPlugin = plugin;
+      } else {
+        throw new Error(`[Embroider] a babel plugin without an absolute path was from: ${plugin}`);
+      }
+
+      const pkgPath = resolvePackagePath.findUpPackagePath(absolutePathToPlugin);
+      plugins[absolutePathToPlugin] = pkgPath ? readJSONSync(pkgPath).version : undefined;
```

Comment:
I thought we had decided to make the scenario where we can't find a `pkgPath` an error condition here?

Reply:
Hmm, looks like `maybeNodeModuleVersion` does that, can you update this to use `maybeNodeModuleVersion` to get the `version`?

Reply:
forgot this scenario, fixing.